docs/docs/concepts/server-mcp.md

```diff
@@ -6,7 +6,7 @@ hide:
   - tags
 ---
 
-# MCP Endpoint
+# MCP endpoint in LangGraph Server
 
 The **Model Context Protocol (MCP)** is an open protocol for describing tools and data sources in a model-agnostic format, enabling LLMs to discover
```

Comment:
This top section + requirements should probably move to a conceptual page

Also, the last sentence ("The MCP endpoint is available at: ...") should have `/mcp` inlined and not in a code block